hash/megadriv.xml

```diff
@@ -27866,6 +27891,22 @@ Black screen, writes in ROM area then jumps to PC=1a2000 (encryption?)
 		</part>
 	</software>
 
+	<software name="sonic2p12" cloneof="sonic2">
+		<description>Sonic The Hedgehog 2 (prototype 199209xx, pirate)</description>
+		<year>1992</year>
+		<publisher>Sega</publisher>
+		<notes><![CDATA[
+Earlier than the "Beta 4" prototype. No equivalent build has been found in the Sega QA Archive.
+Original scene release which was altered to include a cracktro and bypass the copy protection. Dump originally in smd format, converted to raw binary due to this softlist not supporting smd files.
```

Comment:
It's not `softlist` blame, it's because .smd are intended to be used by https://www.segaretro.org/Super_Magic_Drive which needs a separate softlist altogether.

Reply:
smd is just a different data ordering, you can convert freely between them.

a lot of the prototypes etc. that were dumped years ago were in smd format.

I don't really think there's any point in making a list containing smd dumps, converting them to .bin is the correct approach.

Reply:
"Softlist blame" removed.

Reply:
Gotta love your assertive way of putting whatever you think it should be an universal approach to do things.
https://old.reddit.com/r/everdrive/comments/s8rd8l/smd_roms_not_loading_mega_everdrive_pro/
> EDIT: To be clear, you will need to use UCON64 or something similar to convert the interleaved smd file to a bin/md file to compare the hashes.

Reply:
yes, there are tools to convert between them, there always have been.  It's 16kb sections interleaved, rather than byteswwapped, and a header, but either way, it's a reproducable transformation, there's really no need to have a list full of copier format files when they can be made 100% from the binary files.

https://www.raphnet.net/electronique/genesis_cart/smdform.txt